docs/index.md

```diff
@@ -1485,14 +1507,18 @@ View SuperAgent's [Makefile][superagent-makefile] for reference.
 
 ### Markdown
 
-The "markdown" reporter generates a markdown TOC and body for your test suite.
+**Enable Names:** `Markdown`, `markdown`
+
+The Markdown reporter generates a markdown TOC and body for your test suite.
 This is great if you want to use the tests as documentation within a Github
 wiki page, or a markdown file in the repository that Github can render. For
 example, here is the Connect [test output][connect-test-output].
 
 ### XUnit
 
-The `xunit` reporter is also available. It outputs an XUnit-compatible XML document, often applicable in CI servers.
+**Enable Names:** `XUnit`, `xunit`
+
+The XUnit reporter is also available. It outputs an XUnit-compatible XML document, often applicable in CI servers.
 
 By default, it will output to the console. To write directly to a file, use `--reporter-options output=filename.xml`.
```

Comment:
Further down you find the HTML reporter, please fix this one as well.

Reply:
@juergba 
in docs, HTML reporter is explained that `The HTML reporter is not intended for use on the command-line.`

so, I think you mean that I should add there is no alias with HTML reporter.
Is that correct?

Reply:
No, this is not correct.
Yes, HTML is not intended for CLI, but Mocha browser also has a setup option:
`mocha.setup({ reporter: 'html' });`
So I suppose both aliases `HTML` and `html` are valid names.

Please also fix the broken link you mentioned in #4134.

Reply:
@juergba 
then, which Is better specifying alias is  for `cli` or `browser` or writing just alias?

Reply:
I don't understand your question, but just write alias.